x-pack/plugin/inference/qa/test-service-plugin/src/main/java/org/elasticsearch/xpack/inference/mock/TestDenseInferenceServiceExtension.java

```diff
@@ -94,6 +94,7 @@ public void infer(
             Model model,
             @Nullable String query,
             List<String> input,
+            boolean stream,
```

Comment:
It seems that we have multiple inference service extensions which do not use streaming at all. Why is this?

Reply:
Got it, thanks for clarifying.